src/drive/web/modules/views/Drive/index.jsx

```diff
@@ -231,15 +231,17 @@ const DriveView = ({
                 carbonCopyResult.data.length > 0,
               className: '',
               label: 'carbonCopy',
-              icon: <Icon icon={iconCertified} size={16} />
+              icon: <Icon icon={iconCertified} size={16} />,
+              tooltip: true
```

Comment:
Hmm it seems the tooltip is only for the Header here right? So instead of having `icon` `tooltip` & label why not having only HeaderComponent: <HeaderToolTip ...><Icon...><Label /></Header ?